### ISSUE TITLE
fr:number hidden input to communicate value has no label

### ISSUE THREAD

**Issue by avernet** [orbeon/orbeon-forms#2483]
This is the input that we use to run the format/unformat XPath functions. A solution would be to use:

- An `xf:output` to make the formatted value available to JavaScript.
- An XForms event sent from JavaScript, to send the new value entered by the user.
